packages/in_app_purchase/ios/Classes/FIAPaymentQueueHandler.m

```diff
@@ -66,6 +66,12 @@ - (void)restoreTransactions:(nullable NSString *)applicationName {
   }
 }
 
+- (void)presentCodeRedemptionSheet {
+  if (@available(iOS 14.0, *)) {
+    [self.queue presentCodeRedemptionSheet];
```

Comment:
Can do we if `respondToSelector`, we should print a warning if this method is not available, either due to xcode 11 or ios 13 and below.

Reply:
Added the check. I also kept the original 14.0 check to keep Xcode happy and not show a warning

Reply:
@cyanglaz This did not solve the issue for this check https://github.com/flutter/plugins/pull/3274/checks?check_run_id=2022635051 
Is there a reason why this check still runs on an image with Xcode 11?

Reply:
Ah taking a look again, it is a static analyzer error.
So we actually have already removed the xcode 11 test recently. So if you rebase master, you shouldn't see the error anymore. Could you also please revert the workaround for xcode11 here :)

Reply:
Done. Let's hope it runs through now 🤞